util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java

```diff
@@ -86,6 +86,7 @@ public void textMessage(Reader in) {
   protected void handleMessage(int stream, InputStream inStream) throws IOException {
     OutputStream out = getSocketInputOutputStream(stream);
     ByteStreams.copy(inStream, out);
+    inStream.close();

```

Comment:
nit: how about closing the stream in a try-finally block?

Reply:
done.